.gitignore

```diff
@@ -9,7 +16,8 @@ lcov.info
 # Results of playwright tests with tracing
 test-results/
 
-# Files from playwright installed with npm
+# Files packages installed with npm (if you use it)
 node_modules
-package-lock.json
-package.json
+
+# Posit Connect deployment
+rsconnect-python
```

Comment:
also not sure about this one. how sensitive is this info?
personally i really like to have this info in the repo. instead of wandering through connect, i have close-to-home details about the deployment.

Reply:
I always feel weird to commit files with line `"filename": "/Users/pasza/Projects/InnovationLead/tapyr",`...
But you're right, especially if you work in devcontainer and the path is `/workspaces/project-name`